defaults/main.yml

```diff
@@ -64,6 +64,7 @@ consul_acl_agent_token: agent
 
 # Services #
 consul_agent_services: false
+consul_service_sanity_healthcheck: true
```

Comment:
I think it would be nice to add a comment explaining briefly what behavior has this variable

Reply:
Probably explanation in readme would be better.